code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -10,17 +10,13 @@
 	symptom_delay_min = 1
 	symptom_delay_max = 1
 	var/passive_message = "" //random message to infected but not actively healing people
-	threshold_descs = list(
-		"Stage Speed 6" = "Doubles healing speed.",
-		"Stealth 4" = "Healing will no longer be visible to onlookers.",
```

Comment:
Is the stealth thing old?

Reply:
Yeah, I checked the code, there's nothing that acts to that effect, not to mention a symptom that's only meant to provide common code shouldn't have it's own thresholds